sklearn/metrics/tests/test_common.py

```diff
@@ -832,7 +840,7 @@ def test_raise_value_error_multilabel_sequences(name):
         assert_raises(ValueError, metric, seq, seq)
 
 
-@pytest.mark.parametrize('name', METRICS_WITH_NORMALIZE_OPTION)
+@pytest.mark.parametrize('name', sorted(METRICS_WITH_NORMALIZE_OPTION))
```

Comment:
Is there a reason not to order the dicts?

Reply:
`METRICS_WITH_NORMALIZE_OPTION` is a set. All the sets are sorted, while all the dictionaries become OrderedDicts.